README.md

```diff
@@ -18,6 +18,29 @@
 
 See in progress spec: https://did-coop.github.io/wallet-attached-storage-spec/
 
+### App Identity and Key Management
+
+To use Wallet Attached Storage (to provision data spaces, to create collections,
+and to read and write resources), the application or service using this client
+will need its own cryptographically provable identity (in the form of a
+[DID](https://w3c.github.io/did-core/)).
+
+At minimum, this means that the app or service using this client will need to
+create and manage its own public-private key pair. The challenges and security
+considerations of cryptographic key management are beyond the scope of this
+usage document, but roughly:
+
+* For in browser **client-only Single Page Applications** (SPAs), the app's
+  identity is ephemeral -- essentially a key pair will be generated for each
+  new user session, and stored in the browser's Local Storage.
```

Comment:
```suggestion
  new user session, and stored in the browser.
```

Mentioning "Local Storage" reads like it is a good idea to use `globalThis.localStorage`, when there are better storage options like non-exportable key in indexedDb. We should either not recommend a particular thing like "Local Storage" or include IndexedDB in the recommendations. This does the former.

Reply:
Agreed yeah. I just meant the general idea, not specific mechanism.